packages/meditrak-app-server/src/routes/social/SocialFeedRoute.ts

```diff
@@ -115,7 +115,7 @@ export class SocialFeedRoute extends Route<SocialFeedRequest> {
     });
 
     const hasMorePages = feedItems.length > numberPerPage;
-    const items = await Promise.all(feedItems.slice(0, numberPerPage - 1).map(f => f.getData()));
+    const items = await Promise.all(feedItems.slice(0, numberPerPage).map(f => f.getData()));
```

Comment:
Hooray for unit tests! They found this bug! The second parameter in `slice()` is non-inclusive, which means that the previous code was actually skipping one feed item at the end of each page 🙀